toast-tk-dao/src/main/java/io/toast/tk/dao/service/dao/access/repository/ProjectDaoService.java

```diff
@@ -0,0 +1,41 @@
+package io.toast.tk.dao.service.dao.access.repository;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import org.bson.types.ObjectId;
+
+import com.github.jmkgreen.morphia.query.Query;
+import com.google.inject.Inject;
+import com.google.inject.assistedinject.Assisted;
+import com.google.inject.name.Named;
+
+import io.toast.tk.dao.domain.impl.common.IServiceFactory;
+import io.toast.tk.dao.domain.impl.report.TestPlanImpl;
+import io.toast.tk.dao.domain.impl.repository.ProjectImpl;
+import io.toast.tk.dao.service.dao.common.AbstractMongoDaoService;
+import io.toast.tk.dao.service.dao.common.CommonMongoDaoService;
+import io.toast.tk.dao.service.init.DbStarter;
+
+public class ProjectDaoService extends AbstractMongoDaoService<ProjectImpl> {
+	
+	public interface Factory extends IServiceFactory<ProjectDaoService>{
+	}
+
+	@Inject
+	public ProjectDaoService(
+		final DbStarter starter,
+		final CommonMongoDaoService cService,
+		final @Named("default_db") String default_db,
+		final @Nullable @Assisted String dbName

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)